build.gradle

```diff
@@ -43,9 +43,12 @@ dependencies {
 	implementation group: 'commons-cli',        name: 'commons-cli',          version: '1.3.1'
 	implementation group: 'org.apache.commons', name: 'commons-collections4', version: '4.1'
 	implementation group: 'org.apache.commons', name: 'commons-lang3',        version: '3.6'
+	implementation group: 'org.apache.commons', name: 'commons-text',         version: '1.8'
 	implementation group: 'org.reflections',    name: 'reflections',          version: '0.9.11'
 	implementation group: 'org.apache.poi',     name: 'poi',                  version: '4.1.1'
 	implementation group: 'org.apache.poi',     name: 'poi-ooxml',            version: '4.1.1'
+	implementation group: 'com.google.guava',   name: 'guava',                version: '20.0'
+	
```

Comment:
I think this dependency is not needed. Please drop it.

Reply:
Actually, it is needed for the set operations performed by `AbstractRule#getBody()`. Also, `guava` has always been among the dependencies, but only transitively included by some other library (I think `antlr`). Since I'm using it directly now, I made that exisiting dependency explicit, keeping the already included version.